workers/gossip-server/src/peer_discovery/expiry_window.rs

```diff
@@ -12,10 +12,10 @@ use tokio::sync::{RwLockReadGuard, RwLockWriteGuard};
 
 /// The amount of time other cluster peers are allowed to give liveness
 /// attestations for an expiry candidate
-pub(crate) const EXPIRY_CANDIDATE_WINDOW_MS: u64 = 10_000; // 10 seconds
+pub(crate) const EXPIRY_CANDIDATE_WINDOW_MS: u64 = 15_000; // 15 seconds
```

Comment:
why? this means we expire a peer 1.5x slower, is there a good reason for this change? 10s should be more than enough time for peers to contest an expiry. If anything we should be lowering this

Reply:
now that we don't mark fresh heartbeats when adding a peer, new nodes that are still warming up into gossip are quick to expire discovered peers since they are using heartbeat timestamps from received peer info payloads. this change is to allow for a little more time for new peers to exchange heartbeats w/ peers they discover so they don't preemptively expire them

Reply:
as per internal discussion, will decrease back to 10s and run another test. this should be enough time for heartbeat exchange, and if not, a more appropriate approach would be to disallow expiry proposals during gossip warmup (not implemented in this PR due to complexity)